mempool/src/core_mempool/mempool.rs

```diff
@@ -155,6 +155,7 @@ impl Mempool {
                 priority.as_str(),
             );
             counters::CORE_MEMPOOL_TXN_CONSENSUS_PULLED.observe((prev_count + 1) as f64);
+            counters::CORE_MEMPOOL_TXN_CONSENSUS_PULLED_BY_BUCKET.with_label_values(bucket.as_str()).observe((prev_count + 1) as f64);
```

Comment:
Let's just replace `CORE_MEMPOOL_TXN_CONSENSUS_PULLED`?

Reply:
If we do that, then the forge tests will have the "bucket" parameter, whereas the current devnet, testnet and mainnet won't have this parameter. So, the grafana plots could get confusing for a while?

Reply:
If that's not an issue, then I can just change the `CORE_MEMPOOL_TXN_CONSENSUS_PULLED` counter.

Reply:
Yeah I think we can get around this issue by just having one dashboard for each, then remove the old one once this is fully rolled out